reference/SimpleCalc/README.md

```diff
@@ -1,10 +1,7 @@
-## Updating SimpleCalc Source Code
+# Simple Calculator
 
-There is a shared source code for all the SimpleCalc solutions that are in the `resources` folder. To update the source code, uncomment the `Import` element in the `SimpleCalculator.csproj` for the solution you want to make changes to. When the solution is built the relevant shared resources will be copied into the solution.
+The Simple Calc workshop is here to help you get started building an app for Desktop, Mobile, and Web with Uno Platform. This workshop is designed to help you get your developer environment set up to start building your first app, while learning about the tools, libraries, and patterns that will help you to build your apps with the Uno Platform.
 
-  <!-- Disabling source code updating - uncomment and do build to update source code from shared resources -->
-  <!-- <Import Project="..\..\resources\shared-resources.targets" Condition="Exists('..\..\resources\shared-resources.targets')" /> -->
+This app was developed using Uno Platform and draws inspiration from [@adolfomarinucci's](https://x.com/adolfomarinucci) [Calculator project.](https://github.com/adospace/reactorui-maui/tree/main/samples/Calculator)
 
-Make changes to the shared source code in the `resources` folder and then build the solution. The shared source code will be copied into the solution. Commit the changes to the solution and the shared source code.
-
-Don't forget to update the other solutions that will be affected eg if you make changes to the XAML, make sure both the XAML solutions are updated.
\ No newline at end of file
+![SimpleCalculator light Image](simplecalculator-light.png) ![SimpleCalculator dark Image](simplecalculator-dark.png)
```

Comment:
Can we also add the workshop link (https://platform.uno/docs/articles/external/workshops/simple-calc/README.html) as we did with TubePlayer [here](https://github.com/unoplatform/Uno.Samples/blob/feature/singleproject/reference/TubePlayer/README.md?plain=1#L43-L45) ?